### ISSUE TITLE
[BUG] Unable to get corrupted records

### ISSUE THREAD

**Issue by abinesh1006** [crealytics/spark-excel#709]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Corrupted record colum value is coming as TODO:  How to show the corrupted record?

### Expected Behavior

Corrupted row values should come

### Environment

```markdown
- Spark version:
- Spark-Excel version:
- OS:
- Cluster environment
```

**Comment by hardik-sehgal** [comment]
any update on this?

**Comment by nightscape** [comment]
No... Can you please copy and paste the questions in the original question and fill them out properly?
